floors/fetcher.go

```diff
@@ -309,6 +309,10 @@ func validateRules(config config.AccountFloorFetch, priceFloors *openrtb_ext.Pri
 		return errors.New("skip rate should be greater than or equal to 0 and less than 100")
 	}
 
+	if priceFloors.Data.FetchRate != nil && (*priceFloors.Data.FetchRate < dataRateMin || *priceFloors.Data.FetchRate > dataRateMax) {
+		return errors.New("FetchRate should be greater than or equal to 0 and less than or equal to 100")
```

Comment:
Nitpick: In case the value of the `dataRateMin` and `dataRateMax` constants changes someday, should we use their values in the error message instead?
```
312     if priceFloors.Data.FetchRate != nil && (*priceFloors.Data.FetchRate < dataRateMin || *priceFloors.Data.FetchRate > dataRateMax) {
313 -       return errors.New("FetchRate should be greater than or equal to 0 and less than or equal to 100")
    +       return fmt.Errorf("FetchRate should be greater than or equal to %d and less than or equal to %d", dataRateMin, dataRateMax)
314     }
315
floors/fetcher.go
```

Reply:
Very less chance of constant getting changed as these are % based values which will have range 0 to 100 always.